scripts/retry.js

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env node
+
```

Comment:
Why don't we use typescript for this file?

Reply:
This is explained in the comment below.

The problem is that to use typescript we have to run `npm install` first. However, there are cases where we'd like to use retry without installing dependencies. Including wrapping `npm install` command itself as it tend to flake from time to time due to network glitches.

Reply:
The alternative that I was considering was a bash script. But I figured that dependency-less pure JS is easier for us to write/read/understand.